src/ucp/api/ucp_def.h

```diff
@@ -771,5 +772,22 @@ typedef struct ucp_ep_params {
  */
 #define UCP_ENTITY_NAME_MAX 32
 
+/**
+ * @ingroup UCP_TRANSPORT
+ * @brief Maximum number of transport/device name pairs
+ */
+#define UCP_MAX_TRANSPORTS 10
+
+/**
+ * @ingroup UCP_ENDPOINT
+ * @brief Array of transport layers and device names used by an endpoint
+ */
+typedef struct ucp_transports {
+    unsigned int num_entries;
```

Comment:
1. we'll need field_mask for this struct as well
2. "unsigned" instead of "unsigned int"
3. pls align struct fields by column and add documentation for them

Reply:
I fixed points 2 and 3. I don't understand the first point about field_mask. I see other structures in uct_defs.h do this so that additional fields can be added to the attribute and query structures as UCX adds new function. In this case, these structures are fixed output data from UCX, not something that the user sets flags to specify certain attributes are used or are to be queried.